package.json

```diff
@@ -7,9 +7,13 @@
     "node-fetch": "^2.6.1"
   },
   "devDependencies": {
-    "@types/node-fetch": "^2.5.12"
+    "@types/node-fetch": "^2.5.12",
+    "typescript": "^4.4.2"
   },
   "scripts": {
+    "build": "tsc",
+    "prestart": "npm run build",
```

Comment:
`prestart` will automatically run before `start` so you don't have to remember to build each time.